packages/gestalt/src/Checkbox.css

```diff
@@ -20,22 +32,18 @@
 
 .checkDarkGray {
```

Comment:
since these css classes no longer set anything else besides composing the background color from `Colors.css` can we just update the js code to directly import those styles and have the logic directly set those classnames? this will reduce the duplication in my opinion